src/appleseed/renderer/modeling/camera/camera.cpp

```diff
@@ -124,20 +124,29 @@ Camera::~Camera()
 
 bool Camera::on_render_begin(
     const Project&          project,
+    const BaseGroup*        parent,
+    OnRenderBeginRecorder&  recorder,
     IAbortSwitch*           abort_switch)
 {
-    m_shutter_open_begin_time = m_params.get_optional<float>(
-        "shutter_open_begin_time",
-        ShutterOpenBeginTimeDefaultValue);
-    m_shutter_open_end_time = m_params.get_optional<float>(
-        "shutter_open_end_time",
-        m_shutter_open_begin_time);
-    m_shutter_close_end_time = m_params.get_optional<float>(
-        "shutter_close_end_time",
-        ShutterCloseEndTimeDefaultValue);
-    m_shutter_close_begin_time = m_params.get_optional<float>(
-        "shutter_close_begin_time",
-        m_shutter_close_end_time);
+    m_shutter_open_begin_time =
+        m_params.get_optional<float>(
+            "shutter_open_begin_time",
+            ShutterOpenBeginTimeDefaultValue);
+
+    m_shutter_open_end_time =
+        m_params.get_optional<float>(
+            "shutter_open_end_time",
+            m_shutter_open_begin_time);
```

Comment:
It's inverted. You name it ```end_time``` but you pass ```begin_time``` as argument.
Just to be sure it's the right behaviour

Reply:
I haven't touched that code, just reformatted it. That said, it's correct. The second argument is the default value. When no open end time is specified, we use the open begin time.

Reply:
Yep, it's what I was thinking.
Just to be sure.